### DISCUSSION TITLE
User guide \ Switching between kanas

### DISCUSSION THREAD

**Discussion by fref** [google/mozc#895] [Q&A]
Hello, 
I recently installed Mozc on my personal computer and my work laptop. Switching between roman and kana types works ok on my laptop (using alt-capslock and ctrl-capslock).
On my desktop, though, this does not work. It's probably because I have a logitech gaming keyboard (g910).

I can't find the documentation about the meanings of the various "modes" which are available in the keymap customization or find a way to have the capslock shortcuts to work. 

Any suggestions are welcome.

**Comment by hiroyuki-komatsu** [comment]
I guess alt-capslock and ctrl-capslock are shortcuts managed by IMF (input method framework, like Ibus, Fcitx, etc.) but not Mozc.

You might want to check the IMFs on your laptop and desktop.

**Comment by fref** [comment]
Both Ibus, Fcitx are Linux IMF, I'm on Windows 10

**Comment by hiroyuki-komatsu** [comment]
In that case, to make sure if it's Mozc specific issue or OS level configuration issue, please check if the system Japanese Input works as expected.
